utils.js

```diff
@@ -0,0 +1,11 @@
+import * as fs from 'fs'
+export function getAuthData(){
```

Comment:
This function reads file each time we want to get auth data. We could read it once and store in a variable. It would look like this: 
`const authData = JSON.parse(fs.readFileSync(filePath));`
Please let me know in case this doesn't work.